server/datastore/memorydb/index.ts

```diff
@@ -64,4 +64,14 @@ export class InMemoryDatastore implements Datastore {
     this.posts.splice(index, 1);
     return Promise.resolve();
   }
+
+  getLikes(postId: string): Promise<number> {
+    const likes = this.likes.filter(x => x.postId === postId).length;
+    return Promise.resolve(likes);
+  }
+
+  isDuplicateLike(like: Like): Promise<boolean> {
+    const isExists = this.likes.indexOf(like) > 0 ; 
```

Comment:
`>= 0`, otherwise we won't return true if it's a duplicate with the first like.

Reply:
good catch